corehq/apps/es/users.py

```diff
@@ -107,12 +108,24 @@ def _from_dict(self, user_dict):
         user_dict['user_data_es'] = []
         if user_dict.get('base_doc') == 'CouchUser' and user_dict['doc_type'] == 'CommCareUser':
```

Comment:
Exciting!  I think you can drop the second half of this conditional now - that'll be what's needed to get web users populated properly

Reply:
I have noticed that the webuser user_data updates are still not working [per this test you have added](https://github.com/dimagi/commcare-hq/pull/33688/files#diff-7cfcbcfb34b82f54d196ac035c3839a8b61a164ea16efe51fc59fcea7e1369daR73). Are you planning to fix this in your PRs? I suppose that would be required to add Webuser tests in this PR (with multiple domain memberships)